### DISCUSSION TITLE
用了5.0以后超过一天的延时消息该怎么实现呢

### DISCUSSION THREAD

**Discussion by pc7364** [apache/rocketmq#8305] [General]
用了5.0以后超过一天的延时消息该怎么实现呢

**Comment by mxsm** [comment]
@pc7364  可以看一下https://rocketmq.apache.org/zh/docs/featureBehavior/02delaymessage  这个延迟消息。TImerMessage

**Reply by pc7364** [reply]
我想发的是延时时间超过一天的消息

**Reply by good-134** [reply]
重新投递
